salt/states/svn.py

```diff
@@ -95,13 +95,27 @@ def latest(name,
                      'The path "{0}" exists and is not '
                      'a directory.'.format(target)
                      )
+    try:
+        current_info = __salt__['svn.info'](cwd, target, user=user, username=username, password=password, fmt='dict')
+        svn_cmd = 'svn.update'
+    except exceptions.CommandExecutionError:
```

Comment:
If we catch this, won't `svn_cmd` not be updated and we'll do the wrong thing later on?